dashboard/controllers/authController.go

```diff
@@ -1,15 +1,14 @@
 package controllers
 
 import (
-	"net/http"
-	"os"
-	"time"
-
 	"github.com/gin-gonic/gin"
 	"github.com/golang-jwt/jwt/v5"
-	"github.com/rohansingh9001/Neura-Launch-Dashboard/initializers"
-	"github.com/rohansingh9001/Neura-Launch-Dashboard/models"
+	"github.com/rohansingh9001/Neura-Launch-Dashboard/dashboard/initializers"
+	"github.com/rohansingh9001/Neura-Launch-Dashboard/dashboard/models"
 	"golang.org/x/crypto/bcrypt"
+	"net/http"
+	"os"
+	"time"
```

Comment:
what formatter are you using?

Reply:
This was required for the refactor. Since the go.mod is now in the root dir as opposed to the /dashboard dir

Reply:
yea not this... I'm trynna figure out what formatter pushed the net/http, os and time imports at the bottom